### ISSUE TITLE
Retain Fastg output from Spades

### ISSUE THREAD

**Issue by samhunter** [ibest/ARC#54]
Spades generates a FASTG format file as well as FASTA for contigs. Some users would appreciate having the FASTG contigs available to them.

**Comment by quxiaojian** [comment]
Yeah. Thank you Sam. Really I could realize this by adding the parameter keep_assemblies=True. If anyone could keep only the Fastg file, it would be better. Thanks.
XJ
